web/src/routes/stores/[...slug].svelte

```diff
@@ -0,0 +1,25 @@
+<svelte:head>
+    <title>Stores</title>
+</svelte:head>
+
+<script context="module">
+    export async function load({ params }) {
+        const { slug } = params;
+        return { props: { storeName: slug } }
+    }
+</script>
+
+<script lang="ts">
+    import { Auction, fromJson } from "$lib/types/auction";
+    import StoreView from "$lib/components/StoreView.svelte";
+    export let storeName: string;
+    let auctions: Auction[] | null = [];
```

Comment:
This component need not be aware of the auctions needed by the child component.

The only case we do that is in the AuctionList, because we take certain decisions (whether to show the "new" button).